src/Dibi/Drivers/MySqlDriver.php

```diff
@@ -139,7 +139,7 @@ public function connect(array & $config)
 	 */
 	public function disconnect()
 	{
-		mysql_close($this->connection);
+		@mysql_close($this->connection);
```

Comment:
Could you add simple description? `@mysql_close($this->connection); // @ - connection can be already disconnected